### ISSUE TITLE
New note unexpectedly inherits previous note's content

### ISSUE THREAD

**Issue by obiyuta** [inkdropapp/forum#64]
## Bug report

If some texts are saved on the clipboard when creating a new note, its contents will be copied to the new note by default.

<img width="796" alt="console" src="https://user-images.githubusercontent.com/3177114/29151260-f29402aa-7dba-11e7-89e5-7ce6349f2f73.png">

### Info
- Platform: (macOS)
- Platform version: (Sierra/10/12.05)
- App Version: (3.7.2)

### Reproduce

- Copy a note contents (select text > ⌘c)
- Create the new note (⌘n)
- The new note is created with clipboard contents

**Comment by craftzdog** [comment]
hi, thank you for the bug report!
Tried to reproduce it but couldn't.
I guess the problem depends on the content of the note.
Does this occur on other note?

**Comment by obiyuta** [comment]
@craftzdog 

I guess the problem doesn't depend on the content.

It seems to occur only when creating from the shortcut (⌘n).
I tried it several times, and it was occurred on every notes i tried to create from the shortcut.
But it didn't occur when creating from the create button (✎).

I reproduced and captured the problem here.

![kapture](https://user-images.githubusercontent.com/3177114/29240494-9dfa3532-7fa1-11e7-98ac-24c04211c9c4.gif)
